gemelli/rpca.py

```diff
@@ -579,9 +579,12 @@ def frequency_filter(val, id_, md):
         return (np.sum(val > 0) / n_samples) > (min_feature_frequency / 100)
 
     # filter and import table for each filter above
-    table = table.filter(observation_filter, axis='observation')
-    table = table.filter(frequency_filter, axis='observation')
-    table = table.filter(sample_filter, axis='sample')
+    if min_feature_count is not None:
+        table = table.filter(observation_filter, axis='observation')
```

Comment:
Note these operations are inplace by default. If that is not expected, suggest either creating a copy prior, or passing `inplace=False`

Reply:
Good to know. I added `inplace=False` just for continuities sake. Thanks!